pkg/host/machine_info_linux_test.go

```diff
@@ -413,3 +418,10 @@ func TestGetGlobsInfo(t *testing.T) {
 		t.Fatal(diff)
 	}
 }
+
+func longScanner(r io.Reader) *bufio.Scanner {
+	const newMaxTokenSize = 1 * 1024 * 1024
+	s := bufio.NewScanner(r)
+	s.Buffer(make([]byte, 4096), newMaxTokenSize)
```

Comment:
s/make([]byte, 4096)/nil/

Reply:
Nice. Read the https://medium.com/@habibridho/golang-nil-vs-empty-slice-87fd51c0a4d.